CHANGELOG.md

```diff
@@ -10,6 +10,21 @@
     # # # for more details. If you did not receive the license, for more information see:
     # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
+*From issue NRLMMD-GEOIPS/geoips#102: 2023-02-22, improve imports*
+
+Refactor
```

Comment:
I am updating this to include both the top level category label headers (=========), as well as second level headers with the actual change summary (-------).  Also updating previous CHANGELOG entry.  This enables informative summaries using the headers in the release notes when building the documentation.  As soon as it lets me push, changes will be in place.